Hunting Queries/CloudAppEvents/SetPolicyConfigInCloudAppEvents.yaml

```diff
@@ -0,0 +1,32 @@
+id: fcd4d774-a0c2-4d12-9e9f-f51dfc310873
+name: 
+description: |
+  "We search for any action type with high frequency that involves adding/modifying/removing something in cloud app policies. We want to see where the properties are modified such that the old value and new value are different, and exclude DisplayName as changing Display Name is a minor property change."
+severity: Medium
+requiredDataConnectors:
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - CloudAppEvents
+queryFrequency: 1h

```

Comment:
some fields are not required for Hunting Queries such as query frequency, triggerthreshold and others. we can remove these. Compare the template from other hunting queries.